core/src/main/java/org/ehcache/core/events/StoreEventSink.java

```diff
@@ -27,7 +29,7 @@
    * @param key removed key
    * @param value removed value
    */
-  void removed(K key, V value);
+  void removed(K key, Invoker<V> value);
```

Comment:
Is there any circumstance where the value presented to/via an event should be subject to delayed materialization?  Could it then be subject to being unavailable when the value is later fetched?